testpilot/frontend/static-src/app/views/experiment-row-view.js

```diff
@@ -69,6 +69,7 @@ export default BaseView.extend({
     this.loggedIn = opts.loggedIn;
   },
 
+// These need to be changed to fit tab mapping: Just Launched, Just Updated, Enabled
```

Comment:
Also, I think we can leave the openDetailPage function alone for right now, since the state really hasn't changed: either you want to see details with the experiment installed, or without the experiment installed.